<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YearsForm_template</class>
 <widget class="QDialog" name="YearsForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="Title of dialog">Years (forms, classes) of students</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="Q3ListBox" name="yearsListBox">
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Q3TextEdit" name="detailsTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addYearPushButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeYearPushButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="modifyYearPushButton">
         <property name="text">
          <string>Modify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sortYearsPushButton">
         <property name="text">
          <string>Sort alphabetically</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="divisionsPushButton">
       <property name="text">
        <string>Divide year automatically by selecting categories</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="activateStudentsPushButton">
       <property name="text">
        <string>Activate all activities for selected students year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deactivateStudentsPushButton">
       <property name="text">
        <string>De-activate all activities for selected students year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>yearsListBox</tabstop>
  <tabstop>detailsTextEdit</tabstop>
  <tabstop>addYearPushButton</tabstop>
  <tabstop>removeYearPushButton</tabstop>
  <tabstop>modifyYearPushButton</tabstop>
  <tabstop>sortYearsPushButton</tabstop>
  <tabstop>divisionsPushButton</tabstop>
  <tabstop>activateStudentsPushButton</tabstop>
  <tabstop>deactivateStudentsPushButton</tabstop>
  <tabstop>closePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
